roles/ceilometer-common/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 public_ipaddr: "{{ hostvars[inventory_hostname][ 'ansible_' + primary_if ].ipv4.address }}"
 internal_ipaddr: "{{ hostvars[inventory_hostname][ 'ansible_' + private_if ].ipv4.address }}"
 ceilometer_bind_host: "{{ internal_ipaddr }}"
+mongodb_servers: "{% for node in groups['controller'] %}{{ hostvars[node]['inventory_hostname'] }}:{{ mongodb_port | default(27017) }}{% if not loop.last %},{% endif %}{% endfor %}"
```

Comment:
Are we after hostname here or IP address?

If hostname should be using `ansible_hostname`
If IP address it should be `['ansible_'+private_if]['ipv4']['address']`